docs/sink-configuration.md

```diff
@@ -78,6 +78,25 @@ The following options are available:
 * `tenant` - Hawkular-Metrics tenantId (default: `heapster`)
 * `labelToTenant` - Hawkular-Metrics uses given label's value as tenant value when storing data
 
+### Kafka
+This sink supports monitoring metrics and events.
+To use the kafka sink add the following flag:
+
+```
+--sink=kafka:<KAFKA_SERVER_URL>[?<OPTIONS>]
```

Comment:
@mvdan ,thanks for your comments so much. The adding lines of Kafka sink are copied from the above sinks to keep pace with their format. So i think these lines are not one line, i will retain this. : )